### PR TITLE
Data Coverage Page

### PR DESCRIPTION
**Pull request by perez-e**
https://github.com/statmuse/statmuse.com/assets/5808659/b6ce103c-2d3d-4dc0-a094-f28ec75f5443

### CODE REVIEW

File: packages/web/src/pages/product/data/[league].astro [hunk 1]
```diff
@@ -0,0 +1,1226 @@
...
+
+const { league } = Astro.params
+
+const meta = {
+  title: 'Data',
+  description:
+    "StatMuse boasts the world's most comprehensive publicly available sports statistics database, with data going back to the inaugural seasons for NFL (1920), NBA (1947), MLB (1876) and NHL (1917).",
+}
+
+const data = [
```
Comment (perez-e) [inline packages/web/src/pages/product/data/[league].astro:24]: @adamelmore should I pull the data into a separate file?

File: packages/web/src/pages/product/data/[league].astro [hunk 2]
```diff
@@ -0,0 +1,1226 @@
+---
+import ProductLayout from '@layouts/product.astro'
+
+export const prerender = true
```
Comment (perez-e) [inline packages/web/src/pages/product/data/[league].astro:4]: @adamelmore I enabled prerender on these pages. Should we do the same for the company about page?

Comment (adamelmore) [reply packages/web/src/pages/product/data/[league].astro:4]: yeah, i think all of these static pages should be prerendered!

File: packages/web/src/layouts/product.astro [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      </div>
+    </div>
+  </Container>
+</Layout>
+<script define:vars={{ prefixPath }}>
+const onChange = (e) => {
+  window.location.pathname = `${prefixPath}/${e.target.value}`
+}
+document.getElementById('select')?.addEventListener('change', onChange)
+</script>
```
Comment (adamelmore) [inline packages/web/src/layouts/product.astro]: do we actually need this JS or can we just use anchor tags?

**Review by adamelmore — COMMENTED** [review]
just a couple notes/questions!

**Pull request merged**